### PR TITLE
Extract fix (v5)

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
It was busted, but not any more!

### CODE REVIEW

**Comment by englercj** [comment]
I feel like some rebasing may need to happen here...

**Comment by GoodBoyDigital** [comment]
eek! yes, that does not look right at all!

**Comment by bigtimebuddy** [comment]
Bam! Just the wrong branch.

**Comment by bigtimebuddy** [comment]
Needs linking, can you run “npm run lintfix”

File: packages/extract/src/webgl/WebGLExtract.js [hunk 1]
```diff
@@ -180,9 +174,12 @@ export default class WebGLExtract
 
         if (renderTexture)
         {
-            textureBuffer = renderTexture.baseTexture._glRenderTargets[this.renderer.CONTEXT_UID];
+            // textureBuffer = renderTexture.baseTexture;//._glRenderTargets[this.renderer.CONTEXT_UID];
```
Comment (bigtimebuddy) [inline packages/extract/src/webgl/WebGLExtract.js]: Is this line okay to remove?

Comment (englercj) [reply packages/extract/src/webgl/WebGLExtract.js]: Yeah with this commented out I don't see where `textureBuffer` is assigned anymore?

Comment (bigtimebuddy) [reply packages/extract/src/webgl/WebGLExtract.js]: @GoodBoyDigital could you take a look at this. Is the comment-out a mistake?

Comment (GoodBoyDigital) [reply packages/extract/src/webgl/WebGLExtract.js]: tweaked! good catch chaps

**Pull request merged**